src/download_stats.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3
+import os
+import urllib.request
+
+from config import *
```

Comment:
`import *` is often considered a bad practice in python, as it may hide invalid imports, wrong variables, etc.